### PR TITLE
Feature/task 20940 initial defendants and charges documents

### PR DESCRIPTION
**Pull request by Alastair-Pandelus**

### CODE REVIEW

File: polaris-pipeline/Common/Dto/Case/DefendantDetailsDto.cs [hunk 1]
```diff
@@ -22,6 +23,32 @@ public class DefendantDetailsDto
         [JsonProperty("dob")]
         public string Dob { get; set; }
 
+        public string Age
+        {
+            get
+            {
+                // YYYY-MM-DD ?
+                if((Dob.Length == 10)
```
Comment (aceltuk) [inline polaris-pipeline/Common/Dto/Case/DefendantDetailsDto.cs:31]: I'll leave this up to you, but you may want to use a Regex expression to ensure it is a valid date before passing the values on to the calculation?

File: polaris-pipeline/Common/Dto/Case/DefendantDetailsDto.cs [hunk 2]
```diff
@@ -22,6 +23,32 @@ public class DefendantDetailsDto
...
+
+                    return age.ToString();
+                }
+
+                return string.Empty;
+            }
+        }
+
         [JsonProperty("youth")]
         public bool isYouth { get; set; }
```
Comment (aceltuk) [inline polaris-pipeline/Common/Dto/Case/DefendantDetailsDto.cs:53]: Case

**Review by aceltuk — COMMENTED** [review]
Just a couple of comments - nothing major.

File: polaris-pipeline/Common/Dto/Case/DefendantDetailsDto.cs (hunk 2 above)
Comment (Alastair-Pandelus) [reply polaris-pipeline/Common/Dto/Case/DefendantDetailsDto.cs]: good spot... fixed

**Review by aceltuk — APPROVED** [review]

**Pull request merged**